sample_apps/simple_app/example_app.mak

```diff
@@ -92,3 +93,7 @@ $(O)/application_enclave.o: $(S)/application_enclave.cc $(I)/application_enclave
 $(O)/cc_helpers.o: $(S)/cc_helpers.cc $(I)/certifier.h $(US)/certifier.pb.cc
 	@echo "compiling cc_helpers.cc"
 	$(CC) $(CFLAGS) -c -o $(O)/cc_helpers.o $(S)/cc_helpers.cc
+
+$(O)/cc_useful.o: $(S)/cc_useful.cc $(I)/cc_useful.h
+	@echo "compiling cc_useful.cc"
+	$(CC) $(CFLAGS) -c -o $(O)/cc_useful.o $(S)/cc_useful.cc
```

Comment:
Compiles just fine:

```
g++ -I../../include -I/usr/local/opt/openssl@1.1/include/ -I../../src/sev-snp/ -O3 -g -Wall -Werror -std=c++11 
-Wno-unused-variable -D X64 -Wno-deprecated-declarations -c -o ./cc_useful.o ../../src/cc_useful.cc
```

Reply:
The other makefiles will also need to be changed in all the sample_apps, the certifier and certifier_test builds and possibly some of the utilities.

Reply:
@jlmucb : The following makefiles are already included, and updated, in this change-set:

```
sample_apps/simple_app/example_app.mak
sample_apps/simple_app_under_app_service/service_example_app.mak
sample_apps/simple_app_under_gramine/gramine_example_app.mak
sample_apps/simple_app_under_sev/sev_example_app.mak
sample_apps/simple_app_under_sev/sev_example_app_with_dummy.mak
```

The existing CI-job `build.yml` already executes make for `certifier.mak`, `certifier_tests.mak`, `cert_utility.mak` and `policy_utilities.mak`. And, also executes `./certifier_tests.exe`. So, we are already covered in this area.

I am extending `build.yml` to also issue `make -f example_app.mak` from `../sample_apps/simple_app` dir. 

This should give us a bit more coverage for this check-in.